charactersheet/charactersheet/viewmodels/dm/encounter_sections/environment_section.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+function EnvironmentSectionViewModel(parentEncounter) {
+    var self = this;
+
+    self.template = 'environment_section.tmpl';
+    self.encounterId = parentEncounter.encounterId;
```

Comment:
I didn't use it. I grabbed it directly from the `CharacterManager` instead. There's no difference functionally.